### DISCUSSION TITLE
Pregunta Módulos, #include y Makefile

### DISCUSSION THREAD

**Discussion by ManuelOvalle** [IIC2133-PUC/2024-2#27] [Tarea 0] [answered]
Hola, tengo una pregunta acerca de los módulos que debemos crear. No sé si debemos crear los struct en módulos .h de forma separada y luego incluirlos en el main.c o simplemente definirlos ahi mismo. En el caso de hacerlo separado me imagino que sus funciones también las creamos en un .c aparte. Sin embargo, no sé si debo alterar el Makefile al hacer esto. En este archivo se menciona que sólo puedo modificar la linea 20, pero me hace ruido no incluir algo al crear otros archivos. Muchas gracias!

**Comment by tomastrivino** [comment, marked as answer]
El modularizar es a elección: hay gente que resuelve la tarea completa en el `main.c`. Sea cual sea la decisión que tomes en cuanto a diseño, **no es necesario editar el Makefile** si agregas otros archivos `.h` y `.c` para efectos de esta tarea.

**Reply by ManuelOvalle** [reply]
Perfecto, gracias!
